src/commands/cross.ts

```diff
@@ -48,7 +47,7 @@ export class Cross {
         try {
             core.startGroup('Installing cross');
 
-            let args = ['install'];
+            const args = ['install'];
```

Comment:
It seems rather unintuitive to me that `const` declaration forbids variable rebinding, but allows the value mutation :|

Reply:
Yeah... :| Just a JS thing. The switch to `const` was an automatic fix though (either from prettier or eslint, but I didn't check).